pkg/apis/autoscaling/register.go

```diff
@@ -16,157 +16,9 @@ limitations under the License.
 
 package autoscaling
 
-import "time"
-
 const (
 	domain = ".knative.dev"
 
 	// InternalGroupName is the internal autoscaling group name. This is used for CRDs.
 	InternalGroupName = "autoscaling.internal.knative.dev"
-
-	// GroupName is the the public autoscaling group name. This is used for annotations, labels, etc.
-	GroupName = "autoscaling.knative.dev"
```

Comment:
We should be keeping GroupName here, I think.
Rest, I guess we can move.

Reply:
I don't think we can keep the GroupName here, because it's used by the other annotation keys, so we will have to import `knative.dev/serving/pkg/apis/autoscaling` into `pkg/autoscaler/config/config.go` and the cyclic dependency issue won't be solved.
```
import cycle not allowed
package knative.dev/serving/cmd/activator
	imports knative.dev/serving/pkg/activator/handler
	imports knative.dev/serving/pkg/activator/util
	imports knative.dev/serving/pkg/apis/serving/v1
	imports knative.dev/serving/pkg/apis/autoscaling
	imports knative.dev/serving/pkg/autoscaler/config
	imports knative.dev/serving/pkg/apis/autoscaling
```

Reply:
would it be the end of the world if we just duplicated GroupName in both places?

Reply:
keep in mind it's 2020, the world's probably ending anyway